release/cloudbuild-deploy.yaml

```diff
@@ -13,39 +13,61 @@
 # To trigger a build automatically, follow the instructions below and add a trigger:
 # https://cloud.google.com/cloud-build/docs/running-builds/automate-builds
 steps:
-# Pull the latest nomulus.jar to local
-- name: 'gcr.io/${PROJECT_ID}/builder:latest'
-  args:
-  - gsutil
-  - cp
-  - gs://${PROJECT_ID}-deploy/${TAG_NAME}/nomulus.jar
-  - .
-# Pull the credential for nomulus tool
+# Pull the credential for nomulus tool.
 - name: 'gcr.io/${PROJECT_ID}/builder:latest'
   args:
   - gsutil
   - cp
   - gs://${PROJECT_ID}-deploy/secrets/tool-credential.json.enc
   - .
-# Decrypt the credential
+# Decrypt the credential.
 - name: 'gcr.io/${PROJECT_ID}/builder:latest'
   entrypoint: /bin/bash
   args:
   - -c
   - |
+    set -e
     cat tool-credential.json.enc | base64 -d | gcloud kms decrypt \
       --ciphertext-file=- --plaintext-file=tool-credential.json \
       --location=global --keyring=nomulus-tool-keyring --key=nomulus-tool-key
-# Deploy spec11 and invoicing pipeline to GCS
+# Deploy the Spec11 pipeline to GCS.
+- name: 'gcr.io/${PROJECT_ID}/nomulus-tool:latest'
+  args:
+  - -e
+  - ${_ENV}
+  - --credential
+  - tool-credential.json
+  - deploy_spec11_pipeline
+# Deploy the invoicing pipeline to GCS.
+- name: 'gcr.io/${PROJECT_ID}/nomulus-tool:latest'
+  args:
+  - -e
+  - ${_ENV}
+  - --credential
+  - tool-credential.json
+  - deploy_invoicing_pipeline
+# Deploy the GAE config files.
+# First authorize the gcloud tool to use the credential json file, then
+# download and unzip the tarball that contains the relevant config files
 - name: 'gcr.io/${PROJECT_ID}/builder:latest'
   entrypoint: /bin/bash
   args:
   - -c
   - |
-    java -jar nomulus.jar -e ${_ENV} --credential tool-credential.json \
-    deploy_spec11_pipeline
-    java -jar nomulus.jar -e ${_ENV} --credential tool-credential.json \
-    deploy_invoicing_pipeline
+    set -e
+    gcloud auth activate-service-account --key-file=tool-credential.json
+    if [ ${_ENV} == production ]; then
+      project_id="domain-registry"
+    else
+      project_id="domain-registry-${_ENV}"
+    fi
+    gsutil cp gs://${PROJECT_ID}-deploy/${TAG_NAME}/${_ENV}.tar .
+    tar -xvf ${_ENV}.tar
+    for filename in cron dispatch dos index queue; do
```

Comment:
Can we put a comment here linking the bug explaining why this won't work with a service account for the time being?

Reply:
Also, if we use "set -e" and this won't work due to service account permissions, will this fail?